### PR TITLE
added FashionMNIST dataset

### PR DESCRIPTION
**Pull request by kashif**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -390,32 +392,32 @@ For example:
 Utils
 =====
 
-make\_grid(tensor, nrow=8, padding=2, normalize=False, range=None, scale\_each=False, pad\_value=0)
+``make\_grid(tensor, nrow=8, padding=2, normalize=False, range=None, scale\_each=False, pad\_value=0)``
```
Comment (soumith) [inline README.rst]: can you either revert this change, or remove the `\` that escapes the underscore.
Right now the entry looks like this:
```
make\_grid(tensor, nrow=8, padding=2, normalize=False, range=None, scale\_each=False, pad\_value=0)
```

File: README.rst [hunk 2]
```diff
@@ -390,32 +392,32 @@ For example:
...
+``scale_each=True`` will scale each image in the batch of images separately rather than
+computing the ``(min, max)`` over all images.
 
-pad_value=<float> sets the value for the padded pixels.
+``pad_value=<float>`` sets the value for the padded pixels.
 
 `Example usage is given in this notebook` <https://gist.github.com/anonymous/bf16430f7750c023141c562f3e9f2a91>
 
-save\_image(tensor, filename, nrow=8, padding=2, normalize=False, range=None, scale\_each=False, pad\_value=0)
+``save\_image(tensor, filename, nrow=8, padding=2, normalize=False, range=None, scale\_each=False, pad\_value=0)``
```
Comment (soumith) [inline README.rst]: same request as above

**Comment by soumith** [comment]
thanks Dr. Kashif!

**Comment by kashif** [comment]
thank you @soumith. In germany my prof. suggested to add the Dr. to my name... in australia we do not do it... perhaps I will change it back 😺
